core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -512,12 +531,17 @@ public void stopPartitions(Set<StopPartition> stopPartitions,
                 .filter(sp -> sp.deleteLocalLog() && topicIdByPartitionMap.containsKey(sp.topicPartition()))
                 .map(sp -> new TopicIdPartition(topicIdByPartitionMap.get(sp.topicPartition()), sp.topicPartition()))
                 .collect(Collectors.toSet());
+
```

Comment:
Can we update the comment in L529? It is no longer true with topic disablement and can confuse the reader:

```
// Note `deleteLocalLog` will always be true when `deleteRemoteLog` is true but not the other way around.
```